Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "tikv-jemallocator"
 # Make sure to update the version in the README as well:
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
The version inconsistency between `tikv-jemallocator` and `tikv-jemalloc-sys` may confuse people.

If we don't intend to make it consistent, I think should list the versions separately in the CHANGELOG.

Reply:
Fixed, now they are consistent.